File: src/test/java/br/com/model/entities/classes/NovaFormaPagamentoTest.java
package br.com.model.entities.classes;

import java.util.Random;

import org.junit.jupiter.api.RepeatedTest;
import org.junit.jupiter.api.Test;

public class NovaFormaPagamentoTest {

    @Test
    @RepeatedTest(5)
    void novaFormaPagamento() {
        System.out.println(FormaPagamento.novaFormaPagamento(new Random().nextInt(5) + 1));
    }
}

File: src/test/java/br/com/model/entities/classes/NovaFormaPagamentoTest.kt
package br.com.model.entities.classes

import br.com.model.entities.classes.FormaPagamento.Companion.novaFormaPagamento
import org.junit.jupiter.api.RepeatedTest
import org.junit.jupiter.api.Test
import java.util.*

class NovaFormaPagamentoTest {
    @Test
    @RepeatedTest(5)
    fun novaFormaPagamento() {
        println(novaFormaPagamento(Random().nextInt(5) + 1))
    }
}